e2e/config/vhd.go

```diff
@@ -44,13 +44,6 @@ var (
 )
 
 var (
-	VHDUbuntu1804Gen2Containerd = &Image{
```

Comment:
did you maske sure to remove all ubuntu 1804 scenarios which were using this VHD?

Reply:
how can I tell ? golang is happy, this var isn't used

Reply:
you're good, just checked myself